src/corelibs/U2Core/src/datatype/primers/PrimerDimersFinder.cpp

```diff
@@ -36,10 +36,15 @@ namespace U2 {
 /* DimerFinderResult */
 /************************************************************************/
 QString DimerFinderResult::getFullReport() const {
-    return QString("<b>Delta</b> G: %1 kcal/mole <b>Base Pairs:</b> %2").arg(deltaG).arg(baseCounts) + "<pre>" + dimersOverlap + "</pre>";
+    return QObject::tr("<b>Delta</b> G: %1 kcal/mole <b>Base Pairs:</b> %2").arg(deltaG).arg(baseCounts) + "<pre>" + dimersOverlap + "</pre>";
```

Comment:
Do not translate on QObject: it can create conflicts for translations from separate modules. Use a dedicated class inside of the logical domain (primers)

Reply:
Replaced with `PrimerStatistict` cuz this is the only translatable class here

Reply:
Хорошо. Предлагаю дальше писать комментарии на русском, чтобы улучшить стиль, чистоту и ясность написанного.